packages/jellyfish-api-core/__tests__/category/account/listAccountHistory.test.ts

```diff
@@ -275,3 +262,88 @@ describe('listAccountHistory', () => {
     expect(history.find((h) => h.type === 'AccountToUtxos' && h.amounts[0] === '-3.97000000@DFI')).toBeTruthy()
   })
 })
+
+describe('listAccountHistory for poolpair', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+  const createToken = createTokenForContainer(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
+    await container.waitForWalletCoinbaseMaturity()
+    await container.waitForWalletBalanceGTE(200)
+    const address = await container.call('getnewaddress')
+    await container.call('utxostoaccount', [{ [address]: '100@0' }])
+    await createToken(address, 'DDAI', 2000)
+    await createPoolPair('DDAI')
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  async function createPoolPair (tokenB: string, metadata?: any): Promise<void> {
+    const poolAddress = await container.call('getnewaddress')
+    const defaultMetadata = {
+      tokenA: 'DFI',
+      tokenB,
+      commission: 0,
+      status: true,
+      ownerAddress: poolAddress
+    }
+    await client.poolpair.createPoolPair({ ...defaultMetadata, ...metadata })
+    await container.generate(1)
+  }
+
+  it('should show AddPoolLiquidity', async () => {
+    const poolAddress = await container.call('getnewaddress')
+    await client.poolpair.addPoolLiquidity({
+      '*': ['10@DFI', '200@DDAI']
+    }, poolAddress)
+    await container.generate(1)
+
+    const histories = await client.account.listAccountHistory()
+
+    expect(histories.find((h) => (h.type === 'AddPoolLiquidity' && h.amounts.includes('-10.00000000@DFI') && h.amounts.includes('-200.00000000@DDAI')))).toBeTruthy()
+  })
+
+  it('should show RemovePoolLiquidity', async () => {
+    const poolAddress = await container.call('getnewaddress')
+    await client.poolpair.addPoolLiquidity({
+      '*': ['10@DFI', '200@DDAI']
+    }, poolAddress)
+    await container.generate(1)
+    await container.call('removepoolliquidity', [poolAddress, '20@DFI-DDAI'])
+    await container.generate(1)
+
+    const histories = await client.account.listAccountHistory()
+
+    expect(histories.find((h) => (h.type === 'RemovePoolLiquidity' && h.amounts.includes('-20.00000000@DFI-DDAI')))).toBeTruthy()
+  })
+
+  it('should show poolSwap', async () => {
```

Comment:
You wrote `should show RemovePoolLiquidity`. 
'R' which is the first letter is Uppercase.
Hence, this should be changed to `should show PoolSwap`

Reply:
Capitalize the `p`.